package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "ixbrl-viewer",
-  "version": "0.0.0",
```

Comment:
Removing this so it isn't misleading to anyone looking at package.json in the git repo. This gets set immediately immediately before publishing, so it does appear in the actual packaged [package.json](https://www.npmjs.com/package/@aaroncameron-wk/ixbrl-viewer/v/1.1.87-7-gd9f38a1?activeTab=code)

Reply:
Does this still work with local installs? `npm install ../ixbrl-viewer`?

Reply:
`install` does, but `pack` doesn't, and I missed that the Dockerfile was already doing a replace on the `0.0.0` version string so I reverted to that. I just didn't love that the version string would always show as `0.0.0` to anyone checking out the repo, but I'm fine with it for now.